golang/main.go

```diff
@@ -0,0 +1,64 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/google/uuid"
+	"github.com/momentohq/client-sdk-go/momento"
+)
+
+func main() {
+	var AuthToken = os.Getenv("MOMENTO_AUTH_TOKEN")
+	const (
+		CacheName             = "momentocache"
+		ItemDefaultTtlSeconds = 60
+	)
+
+	// Initializes Momento
+	client, err := momento.SimpleCacheClient(&momento.SimpleCacheClientRequest{
+		AuthToken:         AuthToken,
+		DefaultTtlSeconds: ItemDefaultTtlSeconds,
+	})
+	if err != nil {
+		fmt.Println(err.Error())
+	} else {
+		// Create Cache and check if CacheName exists
+		err := client.CreateCache(&momento.CreateCacheRequest{
+			CacheName: CacheName,
+		})
+		if err != nil {
+			fmt.Println(err.Error())
+			os.Exit(1)
+		}
+	}
+
+	// Sets key with default TTL and gets value with that key
+	key := []byte(uuid.NewString())
+	value := []byte(uuid.NewString())
+	fmt.Printf("Setting key: %s, value: %s\n", key, value)
```

Comment:
can we switch to basic golang log package for these type messages will make slightly nicer for end user in logs
https://pkg.go.dev/log